docs/get-started.md

```diff
@@ -1,160 +1,65 @@
-Getting Started
-===============
+Prototyping
```

Comment:
I think this file should probably be deleted now.

Reply:
Unless we want to transfer your HTML into Markdown...

That _might_ be worthwhile.